src/main/java/com/google/sps/data/Tutor.java

```diff
@@ -65,6 +65,10 @@ public Float getRating() {
         return (float) this.ratingSum / this.ratingCount;
     }
 
+    public void addAvailabilityByTimeRange(TimeRange timeslot) {
+        this.availability.add(timeslot);
+    }
+
```

Comment:
I think I just wrote the exact function in my "Manage Tutor's Availability" PR but that's totally fine.

Reply:
Oh yeah, I noticed that. I'll delete this before merging.

Reply:
Just realized I can't delete this because it messes up the tests. We might just have to take care of some merge conflicts.